modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -327,16 +329,53 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS).  
+
+When you enable TLS support and generate a connector:
+
+* The connector is upgraded to run on Mule runtime 4.6.
+* TLS is enabled, so that Anypoint Studio displays a form for collecting the connector's required TLS connection configuration information.
+
+To enable TLS, add a `properties.json` file to the root directory of the Design Center API project for your app: 
+
+. In Design Center, Select *Files +* > *New file*.  
```

Comment:
Just noticed the + in *Files +*. Is that correct?

Reply:
Yes, that's how it is in the UI.